resources/assets/js/features/public-profiles/actionTypes.js

```diff
@@ -0,0 +1 @@
+export const GET_DRIVER_PROFILE = 'GET_DRIVER_PROFILE';
```

Comment:
rename this action to more appropriate, this action is going to set state with profile data, it placed in public profile feature component so better name it like PUBLIC_PROFILE_SET_STATE for example: {featureName}_{action or any appropriate description for it}